lib/integration/nation_builder.rb

```diff
@@ -40,16 +40,12 @@ def self.create_rsvp(event_id:, person_id:)
     end
   end
 
+  # Public: fetches list coutns from a fake NB page with json on it.
+  # Nationbuilder page template is only this:
+  # {"supporter_count": {{ settings.supporters_count }}, "volunteer_count": {{ settings.volunteers_count }} }
   def self.list_counts
-    rescue_oauth_errors do
-      response = request_handler(endpoint_path: ENDPOINTS[:lists])
-      list_ids = eval(ENV['NATION_BUILDER_LIST_IDS'])
-      target_lists = response['results'].select{|list| list_ids.values.include?(list['id']) }
-      target_lists.each_with_object({}) do |list, hash|
-        hash_key = list_ids.invert[list['id']]
-        hash[hash_key] = list['count']
-      end
-    end
+    target_page = ENV['NATION_BUILDER_DOMAIN'] + ENDPOINTS[:list_count_page]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.